android/plugins/network/src/main/java/com/facebook/flipper/plugins/network/FlipperOkhttpInterceptor.java

```diff
@@ -228,10 +228,12 @@ private ResponseInfo convertFlipperObjectRouteToResponseInfo(FlipperObject route
     if (TextUtils.isEmpty(requestUrl) || TextUtils.isEmpty(method)) {
       return null;
     }
+    final int statusCode = route.getInt("status");
```

Comment:
I suppose there might be mocks that don't yet have a status set? Should those perhaps default to 200?

Reply:
@passy  I think you are right.  

The Flipper screen allows entry of any string so it could be empty, not a number, or a number that doesn't qualify as an HTTP status code.  So better editing could be done in Flipper.  I was not able to figure out how to do that. (I'm an Android developer and not very skilled in React).

The Android plugin should definitely default to 200 if nothing is entered or if a NumberFormatError is thrown.

I'd like to work on that after the PR is published and see if there are any other improvements.